METS-Programme/openmrs-module-ugandaemr
# ART regimen lines implementation

Reference https://app.asana.com/0/1131715315801874/1198214324308758 

Testing requires to use these PRS:

- UgandaEMR Reports https://github.com/METS-Programme/openmrs-module-ugandaemr-reports/pull/259

To test the PR:

1. Start the UgandaEMR server to install the concepts and program states, once successful restart to move all clients to the First Line Regimen 
2. Select a patient then move them to a new regimen status 
3. When editing the regimen change the type of switch cannot be edited due to side effects 

Updates - 29 July 2021
==================
1. When a patient is enrolled into care, must automatically enroll them into first line Regimen 
2. The Regimen Change form can only be completed when a patient has both an ART Summary and ART Clinical Encounters 

Pending:
1. Data validation rules for regimen switches not being recorded when the regimens change and regimen switch records not matching the regimen change visit dates 
2. ART Regimen Wizard shows the date enrolled and location name which are to be removed see Talk Post https://talk.openmrs.org/t/program-status-dashboard-widget-configuration/33939
